decentraland/screens/ListLandForSaleScreen.js

```diff
@@ -68,13 +68,19 @@ export class ListLandForSaleScreen extends React.Component {
 
     // Note: Conversion to USD will be implemented on v0.2.0
     const manaPerUsd = 30;
-    const priceMana = Number(`${priceInWei}`) / 1e18;
+    // Get mana price using string to avoid imprecise rounding (i.e.: 57999.99999999999)
+    // TODO: Use TasitSDK Utils to dealing with BigNumbers (will be implemented on v0.2.0)
+    const priceManaInWei = `${priceInWei}`;
+    const intPriceManaLength = priceManaInWei.length - 18;
+    const intPriceMana = priceManaInWei.substring(0, intPriceManaLength);
```

Comment:
If we're using the substring method, is it fair to call this an `int`?

Reply:
Good question. I haven't found a good var name to `the integer part of the price`.

Reply:
Maybe `strRoundedPriceMana`?

Reply:
Noted for the next PR.

Reply:
Or `strPriceManaRounded`

Reply:
Resolving this